tests/fail.cmake

```diff
@@ -24,6 +24,10 @@ endif()
 list(APPEND TOIT_SKIP_TESTS
 )
 
+list(APPEND TOIT_FLAKY_TESTS
+  tests/tls-global-cert-test-slow.toit
```

Comment:
I think we have more flaky ones already. I'd add all the ones using badssl.com (3 tests in total):

- `tests/tls-client-cert-test.toit`
- `tests/tls-global-cert-test-slow.toit`
- `tests/tls-test-slow.toit`